src/Scheduler.php

```diff
@@ -27,11 +27,11 @@ public function scheduleStaticJobs(): SchedulerInterface
                     $this->_scheduleStaticJobs();
                 }
                 $this->_getSemaphoreResource(self::SEMAPHORE_RESOURCE_NAME_SCHEDULE)->releaseLock();
-            }catch(\Exception $exception){
+            }catch(\Throwable $throwable){
```

Comment:
non-blocking: if you're feeling particular, the `catch` here could use surrounding spaces

Reply:
haha done =) who even am I anymore?

Reply:
my brain is trying but failing to come up with a joke about "man from space" or some such

go to bed tim

Reply:
![man from space](https://i.pinimg.com/originals/5f/75/50/5f7550b429ccc1b0e0020b1e978551c2.jpg)

(cc @mucha55 )